ci/java-buildpack-dependency-builder.yml.erb

```diff
@@ -0,0 +1,519 @@
+---
+<% resource_names = %w(maven tomcat wildfly your-kit artifactory) %>
+resource_types:
+<% resource_names.each do |resource| %>
+- name: <%= resource %>-resource
```

Comment:
Made a new card because I'd like to get this PR merged.
https://trello.com/c/FbnVsiti/139-update-cf-java-buildpack-dependency-builder-resource-naming-conventions